docs/statistics/cuped.md

```diff
@@ -6,17 +6,53 @@ However, there are options beyond waiting to gather more data, and all these opt
+- filtered results (including segments)
+- explores
+  This is due to the fact that CUPED models are computationally intense and hence we do not want to use them indiscriminately.
 
-## Inner workings
+:::
 
-The statistics behind CUPED quickly get quite involved. If you are looking for more details on our regression based approach, [this paper goes into depth on using regression adjustments for experimental data](https://projecteuclid.org/journals/annals-of-applied-statistics/volume-7/issue-1/Agnostic-notes-on-regression-adjustments-to-experimental-data--Reexamining/10.1214/12-AOAS583.full).
+## Data
+
+When CUPED is enabled, Eppo automatically ingests aggregated data about each subjects in the following ways:
+
+- For each (applicable) experiment metric, we look at the pre-experiment values during a lookback window which defaults to 30 days.
+- Furthermore, we also leverage the [assignment properties](/data-management/properties) for the subject (e.g. country, browser, etc.)
+
+:::note
+As a concrete example, consider an e-commerce website that runs an experiment with 3 metrics:
+
+- total order value (sum of price),
+- number of orders (count of price), and
+- 7-day purchase conversion.
+
+Furthermore, the assignment table has an assignment property indicating the user's country. In this case, our CUPED++ model uses historical data on order value, number of orders, and user country. It does not use historical data on 7-day purchase conversion since it does not have a clear pre-experiment analog.
+:::
+
+Think of the above as the $X$ matrix in a regression; in fact, we will re-use this X to improve estimates for each experiment metric.
+
+## Model
+
+For each of the metrics in the experiment, we now regress the outcome on the covariates as defined above using a ridge regression, separately for each variation.
+We can then use the predictions of these models to improve the effect estimates, with the variance of these estimates being proportional to the mean square error of the predictions.
+Thus, the improvement in the confidence intervals is directly related to how well the pre-experiment data can predict experiment outcomes.
+
+CUPED works best for experiments with long-time users for whom many pre-experiment data points exist. It is generally less effective for newer users; if you are running an experiment on a change in the onboarding flow for new users, there is no prior data to leverage, just the assignment dimensions, and hence there is less benefit. However, one appealing characteristic of our approach that you do not have to worry about CUPED giving you worse results. In the worst case scenario, it does equally well as the standard approach.
+
+## What makes Eppo's CUPED++ different
```

Comment:
This section didn't help me understand how CUPED++ is different. Are the drawbacks you point out related to a basic CUPED model that CUPED ++ does not have? Or are they about our model or CUPED generally?

Reply:
I give two points of difference below; what about it do you not understand?